bin/rucio-atropos

```diff
@@ -34,7 +34,7 @@ def get_parser():
     """
     Returns the argparse parser.
     """
-    parser = argparse.ArgumentParser()
+    parser = argparse.ArgumentParser(description='The Atropos Daemon is responsible for delete the rules with expired eol_at  (end the life at), according to the Lifetime Model. Once the rule is deleted, the replicas covered by the rule will not be protected anymore and eventually, will be deleted by another daemon.')  # noqa E501
```

Comment:
... is responsible for **the deletion of** rules ...
There is also one space too much after eol_at